index.html

```diff
@@ -413,8 +413,8 @@ <h2 class="lead secondary-heading">Packages</h2>
 
       <div class="row">
         <div class="col-12" style="text-align: center">
-          <a class="btn btn-sm btn-outline-danger" href="https://pkg.julialang.org">Package Docs</a>
-          <a class="btn btn-sm btn-outline-danger" href="https://juliaobserver.com/">Julia Observer</a>
+          <a class="btn btn-sm btn-outline-danger" href="https://juliahub.com">JuliaHub: Package Docs and Code Search</a>
+          <a class="btn btn-sm btn-outline-danger" href="https://juliapackages.com/">Julia Observer</a>
```

Comment:
@djsegal: Not sure what you want to call this link—calling it "Julia Packages" is kind of confusing.

Reply:
I see juliaobserver.com is still running. It would have been ideal to shut the old one down and rename the new website to JuliaObserver.com.

Reply:
I'm ok just saying juliapackages.com is Julia Observer in these docs

What's preventing the migration is that JP doesn't have any cron jobs (whereas JO does)

Reply:
Should we then link to juliaobserver.com? If it doesn't have a cron job, won't it fall behind quickly?

Reply:
I'd probably agree with that. The post about shutting down JO was supposed to be an april fools day prank

// JP is still technically in beta till cron jobs get finished

Reply:
What? I thought that was real!